src/Database/LMDB/Simple/Internal.hs

```diff
@@ -169,9 +173,16 @@ instance MonadIO (Transaction mode) where
 -- "Codec.Serialise.Tutorial".
 data Database k v = Db MDB_env MDB_dbi'
 
+-- | FIXME(jdral): this function should be renamed to @peekMDBVal@ to avoid
+-- confusion with the concepts of keys and values that are stored as key-value
+-- pairs in an LMDB database, since both keys and values are both represnted as
+-- @'MDB_val'@s in the lower-level Haskell LMDB bindings.
 peekVal :: Serialise v => Ptr MDB_val -> IO v
 peekVal = peek >=> marshalIn
 
+pokeMDBVal :: Serialise v => Ptr MDB_val -> v -> IO ()
```

Comment:
Would it make sense to add some comments to `pokeMDBVal`?

Reply:
Done. I've added a comment to `peekVal` as well.